packages/venia-ui/lib/components/MiniCart/miniCart.css

```diff
@@ -11,6 +11,7 @@
     visibility: hidden;
 
     /* It animates to being closed. */
+    transform: scale(0);
     transition-duration: 192ms;
     transition-timing-function: var(--venia-global-anim-out);
     transition-property: opacity, visibility;
```

Comment:
Note that `transform` is purposefully omitted from the `transition-property`: we purposefully don't want to animate the growing and shrinking.

Reply:
Wasn't there supposed to be a little bit of translation going on here, though? I'd swear we had something like `transform: translate3d(0, -0.5rem, 0)`.

Anyway, generally I'd like to leave `transform` on the table for animation without known bugs waiting to happen if it's used. Could we just use positioning (`left: 100vw`) or something?